app/models/post.rb

```diff
@@ -1,2 +1,16 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  after_save :recent_comments
```

Comment:
- [ ] Great job creating this callback!  But it should point to `:update_counter` instead of `:recent_comments`.

Like this:

```ruby
  after_save :update_post_counter
```

Right now, it lists the 5 most recent comments every time a new post is created,

If you make this change, it will update the post counter, as intended! 💪